packages/top-app-bar/index.tsx

```diff
@@ -211,16 +230,44 @@ class TopAppBar<T extends {} = HTMLHeadingElement> extends React.Component<
       className,
       dense,
       fixed,
-      title,
       short,
       shortCollapsed,
       prominent,
       scrollTarget,
       tag: Tag,
+      actionItems,
+      navigationIcon,
+      title,
       ...otherProps
       /* eslint-enable no-unused-vars */
     } = this.props;
 
+    /**
+     * @deprecated since 0.11.0. Will be deleted in 0.13.0
+     */
+    /* istanbul ignore if */
```

Comment:
what is the istanbul ignore for?

Reply:
right for codecoverage. It would have to go on a new line I believe. I originally tried to have it in the comment block with deprecated but it was very finicky.  I think it might not be necessary. In this case you would remove everything from the comment above to the end of the if block. In the other instances of `istanbul ignore next` it would be removed with the function itself due to its placement.

Reply:
ok, added a TODO statement noting that the istanbul ignore's should be removed with all deprecated statements and code in v0.13.0.